catalog/leap-ingested/HadISST.yaml

```diff
@@ -0,0 +1,9 @@
+"ncviewjs:meta_yaml_url": "https://github.com/pangeo-forge/HadISST-feedstock/blob/main/feedstock/meta.yaml"
+tags:
+  - climate
+  - oceanography
+  - sea surface temperature
+  - zarr
+stores:
+  - name: hadisst.zarr
+    url: "https://ncsa.osn.xsede.org/Pangeo/pangeo-forge/HadISST-feedstock/hadisst.zarr"
```

Comment:
this is an example of a feedstock for which we are extending the existing meta

Reply:
How would the pyramid-store location be indicated in this schema?